src/components/AppHelmet.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { setSRMessage } from '../actions/ui';
+
+export const AppHelmet = ({ focusTarget, setSRMessage }) => (
+  <Helmet
+    titleTemplate={`%s | ${process.env.REACT_APP_SITE_TITLE}`}
+    defaultTitle={process.env.REACT_APP_SITE_TITLE}
+    onChangeClientState={newState => {
+      setSRMessage(newState.title);
```

Comment:
What do you think of stripping off the ` | TreatmentFinder` text that gets appended to the page title? Seems unnecessary for screen readers.

```suggestion
      setSRMessage(newState.title.replace(` | ${process.env.REACT_APP_SITE_TITLE}`, ''));
```

Reply:
I like having the `| TreatmentFinder` appended to the announcement, just like it is in the `title` element for context, but this is one that could go either way and still be usable.